recipes/easyquery/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "easyquery" %}
+{% set version = "0.1.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/easyquery-{{ version }}.tar.gz
+  sha256: de2872add2fa78c1b827523d124922426d85d37ca2a41041c809f8813b76a6c5
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - numexpr >=2.0
+    - numpy >=1.7
+    - python >=2.7
```

Comment:
It looks like for Python 2.7 to work, you would also need to add a dependency on `future >=0.12`. Unless you really need 2.7 (conda-forge is by default no longer building 2.7 packages), it would be better to make this e.g. `python >=3.6`.

Reply:
You're right! I think it's ok to just support 3.6+ in conda-forge. Thank you for catching this.